src/utils/convert.js

```diff
@@ -400,11 +400,13 @@ const getStepMetaData = (definition, type, key) => {
 
   const hasCustomInputFields = (
     hasInputFieldPreScripting || hasInputFieldPostScripting || hasInputFieldFullScripting ||
-    Boolean(stepDef.custom_fields_url)
+    (type !== 'trigger' && Boolean(stepDef.custom_fields_url))
+    // Triggers don't have custom input fields
```

Comment:
I'd prefer to be clearer and avoid confusion here, by changing this comment to something like: `// Triggers in WB don't have custom input fields`

Reply:
Agreed! Fixed in 746f9de.